scripts/docker_boot.sh

```diff
@@ -77,6 +91,13 @@ CFG_WEBSUBMIT_BIBCONVERTCONFIGDIR = u'/home/invenio/etc/bibconvert/config'
 CFG_WEBSUBMIT_COUNTERSDIR = u'/home/invenio/var/data/submit/counters'
 CFG_WEBSUBMIT_STORAGEDIR = u'/home/invenio/var/data/submit/storage'
 
+CELERY_SECURITY_KEY = u'/home/invenio/ssl/celery/client.key'
+CELERY_SECURITY_CERTIFICATE = u'/home/invenio/ssl/celery/client.pem'
+CELERY_SECURITY_CERT_STORE = u'/home/invenio/ssl/celery/ca.pem'
+CELERY_TASK_SERIALIZER = u'auth'
+CELERY_SECURITY_SERIALIZER = u'msgpack'
```

Comment:
Got it. I mistaked ``CELERY_SECURITY_SERIALIZER`` for ``CELERY_TASK_SERIALIZER``